extensions/exquery/restxq/src/main/java/org/exist/extensions/exquery/restxq/impl/RestXqServiceSerializerImpl.java

```diff
@@ -82,10 +87,55 @@ protected void serializeBinaryBody(final Sequence result, final HttpResponse res
             }
         }
     }
-
+    
     @Override
     protected void serializeNodeBody(final Sequence result, final HttpResponse response, final Map<SerializationProperty, String> serializationProperties) throws RestXqServiceException {
-        try(final DBBroker broker = getBrokerPool().getBroker();
+    	// multipart/related serialization 
+    	if(serializationProperties.get(SerializationProperty.MEDIA_TYPE).equals("multipart/related")){
```

Comment:
Risk for NPE, please swap

Reply:
will be done

Reply:
This [commit](https://github.com/eXist-db/exist/pull/1552/commits/8b6d15bb2c0326fdd9da70eeb6caaf6905f77415) should solve the problem